src/epics_device.c

```diff
@@ -1165,3 +1165,19 @@ DEFINE_DEVICE(stringout, 5, write_stringout);
 DEFINE_DEVICE(mbbi,      5, read_mbbi);
 DEFINE_DEVICE(mbbo,      5, write_mbbo);
 DEFINE_DEVICE(waveform,  5, process_waveform);
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+/*                          Utility Functions                                */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+void dump_db(void)
```

Comment:
Might be worth adding a `FILE *output` parameter, then the caller can pass `stdin` or `stderr` as required; needs a bit more work from the caller as C doesn't do defaults, but would make sense to do anyway.

Reply:
Done in https://github.com/dls-controls/epics_device/pull/5/commits/24323816ff1d83dc5bcc897bc46d73e7adbe4bc1